README.md

```diff
@@ -89,4 +89,4 @@ In order to gather more information, you can use the `tahoma.get_execution_histo
 
 ### Retrieve HomeKit code
 
-If your TaHoma box supports HomeKit natively, the integration will log the HomeKit code on start currently. This is currently an experimental implementation and in order to enable this, you need to have debug logging enabled.
+If your hub (e.g. Somfy TaHoma) supports HomeKit natively, your setup code will be added as a sensor in Home Assistant. Look up your hub in Home Assistant and retrieve the value from the 'HomeKit Setup Code' sensor. You can now configure the [HomeKit Controller](https://www.home-assistant.io/integrations/homekit_controller/) integration in Home Assistant.
```

Comment:
Would be nice to talk about local control and link the current issue within the Core related to Tahoma and Homekit

Reply:
@tetienne I hope core is merged soon, I don't think that many users will face this directly. Do you think that we should add this extra notice? Or just add it when users create an issue.

Reply:
As you said, it should be merged soon.